### PR TITLE
Bug ui form

### LINKED ISSUE #240: [PE-BUG] UI form bug and irregular data format in form
As per issue #198 #199

### PR DESCRIPTION
**Pull request by WilliamRyank**
Fixes #240 
- Set min width of our application higher
- Make the text in the form to be uppercase
- Make the name shown on our app to be uppercase for first letter of each word and lower case for the rest

### CODE REVIEW

**Comment by WilliamRyank** [comment]
Basically if the user input WiLLiaAm RYAn it will be changed into William Ryan

File: src/main/java/organice/model/person/Name.java [hunk 1]
```diff
@@ -28,7 +28,12 @@
     public Name(String name) {
         requireNonNull(name);
         checkArgument(isValidName(name), MESSAGE_CONSTRAINTS);
-        fullName = name;
+        String[] nameArr = name.trim().split(" ");
+        String fullName = "";
+        for (String str : nameArr) { //Make the first character in name uppercase the rest lowercase
+            fullName += str.substring(0, 1) + str.substring(1).toLowerCase() + " ";
+        }
+        this.fullName = fullName.trim();
```
Comment (TheRealRavager) [inline src/main/java/organice/model/person/Name.java]: Mr Francesco de Freta will be wrongfully uppercased to Francesco De Freta while Dr Ronald McDonald will have his name changed to Ronald Mcdonald. I think we should leave this part to the users to input properly.

File: src/main/java/organice/ui/CommandBox.java [hunk 2]
```diff
@@ -61,13 +61,6 @@ private void setStyleToIndicateCommandFailure() {
         styleClass.add(ERROR_STYLE_CLASS);
     }
 
-    /**
-     * Sets the focus to the command box so that user can type in the next command.
-     */
-    public void requestFocus() {
-        commandTextField.requestFocus();
-    }
-
```
Comment (TheRealRavager) [inline src/main/java/organice/ui/CommandBox.java]: What does this do and why has it been removed?

Comment (C-likethis123) [reply src/main/java/organice/ui/CommandBox.java]: ^Same thoughts as @TheRealRavager

**Pull request merged**